routes/userRoutes.js

```diff
@@ -16,6 +16,19 @@ const createToken = require('../utils/createToken');
  */
 
 routes.get('/', async (req, res, next) => {
+  if (req.query.hotel_id) {
+    const hotelId = req.query.hotel_id;
+    const hotel = await models.Hotel.findById(hotelId);
+    if (hotel) {
+      // if hotel id exists, find all users with that hotel ID and return them
+      const hotelUsers = await models.User.where({hotel_id: hotelId});
+      // filter only hotel staff
+      const hotelStaff = hotelUsers.filter(
+        user => user.user_type !== USER_TYPES.GUEST
+      );
+      res.status(200).json(hotelStaff);
+    }
+  }
```

Comment:
Good idea to move it to this endpoint and to use query string.

The whole `if` statement must go inside the `try catch` block though, so we don't have any unresolved Promise rejections in case something happens to the DB connection when fetching the `Hotel` or `User` models

Reply:
```
try {
    if(req.query.hotel_id) { ... }
    else { ...the original GET endpoint content ... }
} catch { ... }
```